src/main/java/com/faforever/client/chat/event/UnreadPrivateMessageEvent.java

```diff
@@ -0,0 +1,23 @@
+package com.faforever.client.chat.event;
+
+import com.faforever.client.chat.ChatMessage;
+
+public class UnreadPrivateMessageEvent {
+
+
+  private final ChatMessage message;
+  private final boolean chatFocused;
+
+  public UnreadPrivateMessageEvent(ChatMessage message, boolean chatFocused) {
+    this.message = message;
+    this.chatFocused = chatFocused;
+  }
+
+  public ChatMessage getMessage() {
+    return message;
+  }
+
+  public boolean isChatFocused() {
+    return chatFocused;
+  }
```

Comment:
Use Lombok's `@RequiredArgsConstructor` and `@Getter` (I recently added Lombok to the client)